### PR TITLE
Design fixes

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

File: src/components/InputListItem.js [hunk 1]
```diff
@@ -34,25 +35,37 @@ export default class InputListItem extends React.Component {
...
             uncheckedIcon,
+            leftIcon,
             ...rest,
         } = this.props;
 
         return (
             <label
-                className={`ListItem has-left-icon ${className}`}
+                className={
+                    `ListItem has-right-icon ${
```
Comment (danni) [inline src/components/InputListItem.js]: Can we keep the individual `${}` on one line?

File: script/iconify [hunk 2]
```diff
@@ -50,6 +50,15 @@ files.forEach((path) => {
         }
         );
 
+    let allowColoring = "allow-coloring"
```
Comment (danni) [inline script/iconify]: The meaning of these classes needs explaining in the SASS.

Comment (danni) [inline script/iconify]: Or rename it.

File: src/components/InputListItem.js (hunk 1 above)
Comment (danni) [inline src/components/InputListItem.js]: Oh I see why you've done this now. You can ignore.

File: script/iconify (hunk 2 above)
Comment (DanielHeath) [inline script/iconify]: Renamed.

**Comment by DanielHeath** [comment]
@danni ++?

**Comment by danni** [comment]
Use classnames in iconify.

**Comment by danni** [comment]
++

**Comment by DanielHeath** [comment]
Intermittent test failure in IE with google autocomplete - will look into it if it when I get a chance.

**Pull request merged**